x-pack/plugins/infra/server/lib/sources/sources.ts

```diff
@@ -198,23 +185,12 @@ export class InfraSources {
     return {
       ...updatedSourceConfiguration,
       configuration: mergeSourceConfiguration(
-        staticDefaultSourceConfiguration,
+        defaultSourceConfiguration,
         updatedSourceConfiguration.configuration
       ),
     };
   }
 
-  private async getStaticDefaultSourceConfiguration() {
```

Comment:
this function looked for `sources` field in the kibana infra config and returned it as part of source configuration.  It merges it with "default configuration".  we can remove this function and just get the default configuration since we aren't using `sources` from config in our source configuration anymore.